### PR TITLE
Refactor: Consumables

### PR DESCRIPTION
**Pull request by YvanCywan**
**Describe Pull request**

Reworking of Consumables to enable the easier addition of new items directly through the config. This also has the benefit of reducing line count drastically when there are a large amount of items.

**Questions (please complete the following information):**
- Have you personally loaded this code into an updated qbcore project and checked all it's functionality? [yes] (Be honest)
- Does your code fit the style guidelines? [yes]
- Does your PR fit the contribution guidelines? [yes]

### CODE REVIEW

File: server/consumables.lua [hunk 1]
```diff
@@ -1,64 +1,37 @@
...
-	if not Player.Functions.RemoveItem(item.name, 1, item.slot) then return end
-    TriggerClientEvent("consumables:client:Drink", source, item.name)
-end)
-
-QBCore.Functions.CreateUseableItem("kurkakola", function(source, item)
-    local Player = QBCore.Functions.GetPlayer(source)
-	if not Player.Functions.RemoveItem(item.name, 1, item.slot) then return end
-    TriggerClientEvent("consumables:client:Drink", source, item.name)
-end)
+for cAl, _ in pairs(ConsumablesDrink) do
```
Comment (BerkieBb) [inline server/consumables.lua]: Remove the `, _` for all of these, it's useless

**Review by BerkieBb — CHANGES_REQUESTED** [review]

File: server/consumables.lua [hunk 2]
```diff
@@ -1,64 +1,37 @@
...
-	if not Player.Functions.RemoveItem(item.name, 1, item.slot) then return end
-    TriggerClientEvent("consumables:client:Drink", source, item.name)
-end)
-
-QBCore.Functions.CreateUseableItem("kurkakola", function(source, item)
-    local Player = QBCore.Functions.GetPlayer(source)
-	if not Player.Functions.RemoveItem(item.name, 1, item.slot) then return end
-    TriggerClientEvent("consumables:client:Drink", source, item.name)
-end)
+for cAl in pairs(ConsumablesDrink) do
```
Comment (D4isDAVID) [inline server/consumables.lua]:
```suggestion
for cAl in pairs(ConsumablesAlcohol) do
```
These are alcohol drinks.

**Review by D4isDAVID — CHANGES_REQUESTED** [review]

File: server/consumables.lua (hunk 2 above)
Comment (YvanCywan) [reply server/consumables.lua]: Just spotted that one mate, already handled

**Review by D4isDAVID — APPROVED** [review]
LGTM

**Review by BerkieBb — APPROVED** [review]

**Pull request merged**